packages/node/src/ethereum/block.ethereum.ts

```diff
@@ -72,6 +72,11 @@ export class EthereumBlockWrapped implements EthereumBlockWrapper {
     address?: string,
   ): boolean {
     if (!filter) return true;
+
+    if (filter.to === null) {
+      return transaction.to === null || transaction.to === undefined;
```

Comment:
I don't think this is right. If to is null then it will ignore the filters below no matter what the to field is on the tx